### PR TITLE
Add index.html with links

### PR DESCRIPTION
**Pull request by safranck**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+   <body>
+		<a href="http://girldevelopit.com" target="_blank">
+			<img src ="http://wilt.xyz/pink-logo.png" alt = "Girl Develop It Logo" />
+		</a>
+
+		<h1><a href="http://www.meetup.com/Girl-Develop-It-Los-Angeles/events/223004716/" target+"_blank">HTML &amp; CSS 101</a><br/>
+		by <a href="http://gdila.org">Girl Develop It (gdila.org)</a></h1>
+
+		<img src="images/suzette.jpg" align="left" style="margin-right: 30px; width: 150px; height: auto;"><h3>Instructor:</h3> 
+		<h2><a href="http://suzette.pro">Suzette Franck</a></h2>
```
Comment (pamelafox) [inline index.html]: Can you remove this? We don't want instructor specifics on general slides.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <link rel="stylesheet" type="text/css" href="style.css">
+   </head>
+   
+   <body>
+		<a href="http://girldevelopit.com" target="_blank">
+			<img src ="http://wilt.xyz/pink-logo.png" alt = "Girl Develop It Logo" />
+		</a>
+
+		<h1><a href="http://www.meetup.com/Girl-Develop-It-Los-Angeles/events/223004716/" target+"_blank">HTML &amp; CSS 101</a><br/>
+		by <a href="http://gdila.org">Girl Develop It (gdila.org)</a></h1>
```
Comment (pamelafox) [inline index.html]: Remove gdila.org?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+   <head>
+    <title>HTML and CSS 101 by Girl Develop It</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+   </head>
+   
+   <body>
+		<a href="http://girldevelopit.com" target="_blank">
+			<img src ="http://wilt.xyz/pink-logo.png" alt = "Girl Develop It Logo" />
```
Comment (pamelafox) [inline index.html]: Why's it hosted at wilt.xyz?

File: index.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		<img src="images/suzette.jpg" align="left" style="margin-right: 30px; width: 150px; height: auto;"><h3>Instructor:</h3> 
+		<h2><a href="http://suzette.pro">Suzette Franck</a></h2>
+		
+		<ul>
+			<li>Email: <a href="mailto:hello@suzettefranck.com">hello@suzettefranck.com</a></li>
+			<li>Twitter: <a href="http://twitter.com/suzette_franck" target+"_blank">http://twitter.com/suzette_franck</a></li>
+		</ul> 
+		  
+   		<table id="theTable" align="left">
+		  <tr>
```
Comment (pamelafox) [inline index.html]: To make this table a bit more accessible, I'd suggest a <thead> with cells that say "Class | Slides | PDF | Example | Website | Files"
and then a <td> in each row that says "Class 1", "Class 2", etc.

**Comment by Roenok** [comment]
@safranck Have you had a chance to review the notes on this PR? Thanks!

**Comment by safranck** [comment]
Reviewing now, the logo was on wilt.xyz because the gdi website was redesigned and that logo did not exist in the same location it was indicated to be in.

**Comment by Roenok** [comment]
Hey @safranck! We added an index in the latest revision. Thanks!

**Pull request closed without merging**